pbsmetrics/pbsmetrics.go

```diff
@@ -0,0 +1,350 @@
+package pbsmetrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/rcrowley/go-metrics"
+)
+
+type Metrics struct {
+	metricsRegistry     metrics.Registry
+	RequestMeter        metrics.Meter
+	AppRequestMeter     metrics.Meter
+	NoCookieMeter       metrics.Meter
+	SafariRequestMeter  metrics.Meter
+	SafariNoCookieMeter metrics.Meter
+	ErrorMeter          metrics.Meter
+	InvalidMeter        metrics.Meter
```

Comment:
It looks like the `InvalidMeter` and `CookieSyncMeter` are unused and should be deleted.